_data/epiverse_contributors.csv

```diff
@@ -1,50 +1,54 @@
 "chartgerink",NA,"https://avatars.githubusercontent.com/u/2946344?v=4","code"
 "davidsantiagoquevedo",NA,"https://avatars.githubusercontent.com/u/67650252?u=821e4d145648338b0025bc34304fb9c77a39ab1c&v=4","issue_authors"
 "Deedado71",NA,"https://avatars.githubusercontent.com/u/115372183?v=4","issue_authors"
-"Degoot-AM",NA,"https://avatars.githubusercontent.com/u/19929487?v=4","code"
-"dgmascarina",NA,"https://avatars.githubusercontent.com/u/12283231?v=4","analysis"
+"Degoot-AM",NA,"https://avatars.githubusercontent.com/u/19929487?u=d154322701121d940d28d02e41a5749f8d354274&v=4","issue_authors"
+"dgmascarina",NA,"https://avatars.githubusercontent.com/u/12283231?v=4","research"
 "ekamau",NA,"https://avatars.githubusercontent.com/u/7207204?v=4","issue_authors"
-"epiverse-trace-bot",1,"https://avatars.githubusercontent.com/u/115572122?v=4","code"
 "erees",NA,"https://avatars.githubusercontent.com/u/22442251?v=4","issue_authors"
 "ffinger",3,"https://avatars.githubusercontent.com/u/12323626?v=4","code"
 "fsabrilb",NA,"https://avatars.githubusercontent.com/u/30728976?v=4","issue_authors"
-"GeraldineGomez",38,"https://avatars.githubusercontent.com/u/46997659?v=4","code"
+"GAcero02",NA,"https://avatars.githubusercontent.com/u/119520363?u=5a51c93e6e3da6c845bc6d566f681db08098a60a&v=4","issue_authors"
+"GeraldineGomez",140,"https://avatars.githubusercontent.com/u/46997659?v=4","code"
+"hawyndiaz",NA,"https://avatars.githubusercontent.com/u/150168036?v=4","research"
 "hyolimkang",NA,"https://avatars.githubusercontent.com/u/66198338?u=e8557c295ca2f8b89182e0a3c4d1781e160fd3f7&v=4","issue_authors"
 "jameshay218",NA,"https://avatars.githubusercontent.com/u/9113187?v=4","issue_authors"
 "jamesmbaazam",NA,"https://avatars.githubusercontent.com/u/43169674?v=4","code"
-"jd-otero",195,"https://avatars.githubusercontent.com/u/56242573?v=4","code"
+"jd-otero",31,"https://avatars.githubusercontent.com/u/56242573?v=4","code"
 "johanaatorress",NA,"https://avatars.githubusercontent.com/u/148384716?v=4","issue_authors"
 "joshwlambert",NA,"https://avatars.githubusercontent.com/u/46606402?u=3aa7c369f79a1271a80ffc01781a24db084d1f0e&v=4","issue_authors"
 "Joskerus",102,"https://avatars.githubusercontent.com/u/98434743?v=4","code"
-"jpavlich",10,"https://avatars.githubusercontent.com/u/1485744?v=4","code"
+"jpavlich",11,"https://avatars.githubusercontent.com/u/1485744?v=4","code"
 "juan-umana",NA,"https://avatars.githubusercontent.com/u/64109621?u=b4097abb1d092e3a1e217c6d69a02a27aad5608e&v=4","issue_authors"
 "Juanmontenegro99",NA,"https://avatars.githubusercontent.com/u/60274234?v=4","code"
 "Karim-Mane",NA,"https://avatars.githubusercontent.com/u/84502011?u=228e2c85d83394cc284f3a0ebdcd644e4b926e45&v=4","issue_authors"
 "kcharniga",NA,"https://avatars.githubusercontent.com/u/32932075?v=4","issue_contributors"
-"lgbermeo",21,"https://avatars.githubusercontent.com/u/123580898?v=4","code"
-"macataci",4,"https://avatars.githubusercontent.com/u/88789311?v=4","code"
```

Comment:
We keep randomly losing contributors. I wonder if it's not a rate-limit issues that is hidden by the allcontributors package.

Could you investigate and possibly add a `Sys.sleep()` in the loop in this ends up being the source of the issue

Reply:
Ah no! I noticed it with some others but it appears to change everytime. Inspecting... 😊